test/livebook/intellisense_test.exs

```diff
@@ -504,34 +504,32 @@ defmodule Livebook.IntellisenseTest do
 
       assert [
                %{
-                 label: "join/1",
-                 kind: :function,
-                 detail: ~S{Enum.join(enumerable, joiner \\ "")},
+                 detail: "Date.utc_today(calendar \\\\ Calendar.ISO)",
                  documentation: """
-                 Joins the given `enumerable` into a string using `joiner` as a
-                 separator.
+                 Returns the current date in UTC.
 
                  ```
-                 @spec join(t(), String.t()) :: String.t()
```

Comment:
this is now `join(t, binary) :: binary` on Elixir main so I changed the test while at it.